### DISCUSSION TITLE
an old ckan is in a failed virtual server

### DISCUSSION THREAD

**Discussion by crstn-aiw** [ckan/ckan#7320] [Q&A] [answered]
hello i am new at ckan and linux. i need some help if you have some info?
our old linux virtual server failed and cannot ssh/putty into it. it is outdated and no longer functioning.
there is a backup disk and i used it as reference and database copy to a new ckan latest version and postgresql into the new server but having some issues migrating the data (postgre from original is in 9.3). I copied over the database copy from backup disk and i am able to see it reads the organization and groups but datasets are not loading correctly.  i have tried following as close as i can to documentation from ckan (installing from package). 

Is there a configuration somewhere i may be missing? or what may be best way in dealing with this scenario?

any help is appreciated. thank you

**Comment by pdelboca** [comment, marked as answer]
Hello @crstn-aiw ! It's hard to provide insightful ideas given the complexity of the scenario.

Said that, if you can read organizations and groups but not datasets maybe it is an issue with the indexing. In CKAN, results for the dataset search comes from Solr while for Orgs/Groups comes from the database.

So I will suggest you to:
 - Check if the database has the dataset/resources data.
 - Check that Solr has been properly configured
 - Try rebuilding the search index: https://docs.ckan.org/en/2.9/maintaining/cli.html?#search-index-rebuild-search-index
 
 That should populate Solr with Datasets data for CKAN to consume.
 
 Let me know if this helps!

**Reply by crstn-aiw** [reply]
Hi @pdelboca thanks for your response! I will try this

**Reply by crstn-aiw** [reply]
Rebuilding the search index @pdelboca worked perfectly, thank you so much!
